### DISCUSSION TITLE
Support for PayPal

### DISCUSSION THREAD

**Discussion by darrencoltman** [woocommerce/woocommerce#45073] [WooCommerce, Blocks, and Site Editing]
My current theme shows PayPal and Stripe links on the checkout page but the woocommerce blocks checkout only shows Stripe. Any ideas? Apart from that very nice. Thanks!

**Comment by Aljullu** [comment]
Hi @darrencoltman, thanks for opening this issue! Cart and Checkout blocks are still in development and not all payment gateways have already updated to support them. You can see a list of the support payment methods in this page:

https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/#section-4

Hope it helps, let us know if you have any other questions!
